src/pages/Account/Profile.tsx

```diff
@@ -61,57 +65,98 @@ const Profile: React.FunctionComponent<PropsType> = ({ location }) => {
             <input
               name="partyName"
               className="input-block input-centered"
               placeholder="Your display name"
-              defaultValue="Mona"
-              type="hidden"
+              defaultValue={githubHandle}
               ref={register({
                 required: true,
                 maxLength: DISPLAY_NAME_MAX_CHAR_COUNT,
               })}
               autoComplete="off"
             />
-            {/* <span className="input-info">
+            <p className="input-info">
               This is your display name (max {DISPLAY_NAME_MAX_CHAR_COUNT}{" "}
               characters)
-            </span> */}
-            {/* {errors.partyName && errors.partyName.type === "required" && (
+            </p>
+            {errors.partyName && errors.partyName.type === "required" && (
               <span className="input-error">Display name is required</span>
             )}
-            {errors.partyName &&* errors.partyName.type === "maxLength" && (
+            {errors.partyName && errors.partyName.type === "maxLength" && (
               <span className="input-error">
                 Display name must be {DISPLAY_NAME_MAX_CHAR_COUNT} characters or
                 less
               </span>
-            )} */}
+            )}
+            {/* TODO: Proper text, namings */}
+            <input
+              name="realName"
+              className="input-block input-centered"
+              placeholder="Your real name"
+              defaultValue={realName}
+              ref={register()}
+              autoComplete="off"
+            />
+            <p className="input-info">This is your real name</p>
+            <input
+              name="companyTitle"
+              className="input-block input-centered"
+              placeholder="Your title"
+              defaultValue={profilePrefilledData?.companyTitle}
+              ref={register()}
+              autoComplete="off"
+            />
+            <p className="input-info">This is your title</p>
+            <input
+              name="companyDepartment"
+              className="input-block input-centered"
+              placeholder="Your department"
+              defaultValue={profilePrefilledData?.companyDepartment}
+              ref={register()}
+              autoComplete="off"
+            />
+            <p className="input-info">This is your department</p>
+            <a
+              className="profile-picture-button Profile__summit-snap"
+              href="https://virtual.githubphotobooth.com/virtual/capture/gr99n"
+              {...externalUrlAdditionalProps}
+            >
+              Take a Summit snap
+            </a>
             {user && (
               <ProfilePictureInput
                 venueId={venueId}
                 setValue={setValue}
+                githubHandle={githubHandle}
```

Comment:
Should this also just be a field we set on the user profile? Then we could just read it off `user`?

Reply:
Discussion happening in https://github.com/sparkletown/sparkle/pull/1667#discussion_r659567943

Reply:
I agreed it was fine in https://github.com/sparkletown/sparkle/pull/1667#discussion_r659636177, so this can probably be resolved.